src/components/VideoCapture/index.tsx

```diff
@@ -63,7 +63,10 @@ const initialStateWithoutMediaStream: Omit<State, 'hasMediaStream'> = {
 
 const IDEAL_CAMERA_WIDTH_IN_PX = 1080 // Full HD 1080p
 
-const RECORDING_TIMEOUT_ERRORS_MAP: Record<CaptureMethods, ErrorProp> = {
+const RECORDING_TIMEOUT_ERRORS_MAP: Record<
+  Exclude<CaptureMethods, 'poa'>,
```

Comment:
repeated here.

Reply:
I'll make sure to include this suggestion in the work I'm doing in a related issue.